catalogs/models.py

```diff
@@ -33,17 +42,47 @@ class Catalog(models.Model):
         default="https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/"
         "SP800-53/rev5/json/NIST_SP-800-53_rev5_catalog.json",
     )
+    version = models.CharField(
+        max_length=64,
+        blank=False,
+        default="CMS ARS 3.1",
+    )
+    impact_level = models.CharField(
+        choices=ImpactLevel.choices,
+        max_length=20,
+        blank=False,
+        default=ImpactLevel.MODERATE,
+        help_text="FISMA impact level of the project",
+    )
     created = models.DateTimeField(auto_now_add=True, db_index=True)
     updated = models.DateTimeField(auto_now=True, db_index=True, null=True)
 
     def __str__(self):
         return self.name
 
 
+@receiver(post_save, sender=Catalog)
+def add_controls(sender, instance, **kwargs):
+    if kwargs["created"]:
```

Comment:
Not a blocker, just a preference:
```suggestion
def add_controls(sender, instance, created, **kwargs):
    if created:
```